rules-reviewed/eap7/eap6/resteasy.windup.xml

```diff
@@ -0,0 +1,826 @@
+                    <link title="RESTeasy Client Application Changes"
+                        href="https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/7.0.beta/migration-guide/chapter-4-application-changes" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+            <where param="type">
+                <matches pattern="(ProxyBuilder|ProxyConfig|ProxyFactory)" />
+            </where>
+        </rule>
+        <rule id="resteasy-eap6-000005">
+            <when>
+                <javaclass references="org.jboss.resteasy.client.EntityTypeFactory" />
+            </when>
+            <perform>
+                <hint title="Resteasy Client API EntityTypeFactory deprecation" effort="1" severity="mandatory">
+                    <message><![CDATA[
+                    There is no direct replacement for `org.jboss.resteasy.client.EntityTypeFactory` class.
+                    
+                    `org.jboss.resteasy.client.ClientResponse<T>` is a generic type in the Resteasy client framework,
+                    but `org.jboss.resteasy.client.jaxrs.internal.ClientResponse` in the resteasy-client module is not, so
+                    `EntityTypeFactory` is no longer useful.
+                    ]]></message>
+                    <link title="RESTeasy Client Application Changes"
+                        href="https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/7.0.beta/migration-guide/chapter-4-application-changes" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000006">
+            <when>
+                <javaclass references="org.jboss.resteasy.client.ClientExecutor" />
+            </when>
+            <perform>
+                <hint title="Deprecated ClientExecutor interface" effort="1" severity="mandatory">
+                    <message>Replace the `org.jboss.resteasy.client.ClientExecutor` usage with `org.jboss.resteasy.client.jaxrs.ClientHttpEngine` class.</message>
+                    <link title="RESTeasy Client Application Changes"
+                        href="https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/7.0.beta/migration-guide/chapter-4-application-changes" />
+                    <tag>resteasy</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="resteasy-eap6-000007">
+            <when>
+                <javaclass references="org.jboss.resteasy.client.ClientRequestFactory" />
+            </when>
+            <perform>
+                <hint title="Deprecated ClientRequestFactory class" effort="1" severity="mandatory">
+                    <message>Replace the `org.jboss.resteasy.client.ClientRequestFactory` usage with `org.jboss.resteasy.client.jaxrs.org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder` class.</message>
```

Comment:
`org.jboss.resteasy.client.jaxrs.org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder` is definitely not right :-)